src/org/chartjs/PieDAOChartView.js

```diff
@@ -25,12 +25,15 @@ foam.CLASS({
     },
     {
       class: 'foam.mlang.ExprProperty',
-      name: 'keyExpr',
+      name: 'keyExpr'
     },
     {
       class: 'foam.mlang.ExprProperty',
-      name: 'valueExpr',
+      name: 'valueExpr'
     },
+    {
+      name: 'backgroundColor'
```

Comment:
Give this a default value so you don't need to check it before using it below and so you show an example of what it can take. Can this only be a function or can it also just be an array of colors or something?

Reply:
One or the other. function or array of colours.